legacy/ble.c

```
@@ -258,29 +274,33 @@ void ble_uart_poll(uint8_t *buf) {
         layoutRefreshSet(true);
       }
       break;
+    case BLE_CMD_PAIR_STATE:
+      if (ble_usart_msg.cmd_vale[0] == BLE_PAIR_STATE_SUCCESS) {
+        layoutBlePairSuccess();
+      } else if (ble_usart_msg.cmd_vale[0] == BLE_PAIR_STATE_FAILED) {
+        layoutBlePairFailed();
+      }
+      break;
     case BLE_CMD_PASSKEY:
       if (ble_usart_msg.cmd_len == 0x06) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)